azure-pipelines.yml

```diff
@@ -122,7 +122,7 @@ extends:
                 condition: succeeded()
                 inputs:
                   targetType: inline
-                  script: 'node tools/cli/runAppsWithE2ETests.js --appUrl=https://localhost:4002 --reportFileName=e2e-tests-report-perf --envType=perf'
+                  script: 'pnpm exec ts-node tools/cli/runAppsWithE2ETests.ts --appUrl=https://localhost:4002 --reportFileName=e2e-tests-report-perf --envType=perf'
```

Comment:
Is this a built-in function of pnpm? Where did ts-node come from?

Reply:
Ah I see, it was already present in package.json.

Reply:
`pnpm exec` is the pnpm equivalent of `npx`, so that part is built-in to pnpm. `ts-node` is a package used for running TS-based node scripts. Since this script line runs in the other repo where the host package lives, ts-node is something I added to that repo recently for this purpose. I'll share the commit info with you privately.